app/src/main/res/layout/activity_friend_item.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@drawable/background"
+    tools:context=".FriendItemActivity">
+
+    <!-- Toolbar -->
+    <include layout="@layout/toolbar"/>
+
+    <!-- Remove Friend Button -->
+    <ImageButton
+        android:id="@+id/removeFriendButton"
+        style="@style/IconButton"
+        android:layout_marginTop="22dp"
+        android:layout_marginEnd="16dp"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        android:contentDescription="@string/description_remove_friend_button"
+        android:gravity="center"
+        android:onClick="removeFriendButton"
+        app:srcCompat="@drawable/ic_delete" />
+
+    <!-- Info text container -->
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        app:layout_constraintTop_toBottomOf="@id/toolbar"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        android:layout_marginTop="8dp"
+        android:layout_marginStart="16dp"
+        android:orientation="vertical">
+
+        <!-- Points -->
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:orientation="horizontal">
+
+            <TextView
+                style="@style/LargeDarkGreenText"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="@string/points__mountain_category" />
+
+            <TextView
+                android:id="@+id/pointText"
+                style="@style/LargeDarkGreenText"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="120" />
```

Comment:
Cirrus complains about hard coded string. Is it necessary?

Reply:
It was not, indeed. Thanks!